tensorflow/lite/micro/testing/micro_test.h

```diff
@@ -153,6 +155,23 @@ extern tflite::ErrorReporter* reporter;
     }                                                                         \
   } while (false)
 
+#define TF_LITE_MICRO_EXPECT_NEAR_COUNT(x, y, count, epsilon, max_errs)      \
```

Comment:
This is exactly how this function differs from the already existing ones. It does allow for a small percentage of values to go off of the tolerance boundaries. This happens for example in quantized TanH where in order to have zero errors, the tolerance needs to be set way too high because of fixed point arithmetic problems, so instead we allow some fixed number of values to be wrong. I understood @petewarden was ok with the change, please let me know if I need to change it

Reply:
There's no need for this function when quantizing using scale/offset rather than min/max. Removed.